packages/pharos/src/components/toast/pharos-toaster.ts

```diff
@@ -85,6 +94,12 @@ export class PharosToaster extends PharosElement {
     if (toast) {
       this.removeChild(toast);
     }
+    for (const element of this.returnElements) {
+      if (element) {
+        element.focus();
+        break;
+      }
```

Comment:
It might feel better to move this to its own named function, and then you can use `return` instead of `break`.

I think we also need a fallback, returning focus to the main content of the page if no elements were specified or found. Unless that's the default behavior?

Reply:
Originally I was thinking the consumer will pass in the main content of the page in the array as a backup plan, and that's why it takes the input as an array.

Reply:
`if (element)` doesn't seem to be validating a whole lot here other than that `element` is truthy.

Reply:
this might be a more useful check to validate the node is still present in the document
https://developer.mozilla.org/en-US/docs/Web/API/Node/isConnected

Reply:
@jialin-he because consumers don't currently pass this prop at all, I suspect there will be a period where no elements are being passed and we will want to maintain the current behavior in the interim. Does this code already do that?

Reply:
Yes, if no return element is passed, it maintains the current behavior.